daemons/upload-checksum-daemon/Makefile

```diff
@@ -35,3 +35,6 @@ build:
 deploy: build
 	aws s3 cp $(ZIP_FILE) s3://org-humancellatlas-upload-checksum-lambda-deployment/$(DEPLOYMENT_STAGE)/$(ZIP_FILE)
 	aws lambda update-function-code --function-name dcp-upload-csum-$(DEPLOYMENT_STAGE) --s3-bucket org-humancellatlas-upload-checksum-lambda-deployment --s3-key $(DEPLOYMENT_STAGE)/$(ZIP_FILE)
+	# update lambda env vars without calling terraform apply
+	./set_env_vars.py --function-name dcp-upload-csum-${DEPLOYMENT_STAGE} --upload-service-version ${UPLOAD_SERVICE_VERSION}
```

Comment:
I think there is a gotcha here. If you update the lambda env variable with a new version outside of terraform in this current setup, terraform plan will always show expected changes in the next run. As I see it there are 3 options:
1) Populate the version in a secret that terraform does not control. You can then retrieve and log it in the lambdas.
2) Update terrraform.tfvars during deployment process
3) Use a external datasource to fetch current deployment and pass that into lambda env variable definition in terraform.

My vote here is to not have this as a terraform/os environment variable since we don't deploy via terraform. A separate secret seems cleaner.

@sampierson thoughts?

Reply:
https://github.com/HumanCellAtlas/upload-service/pull/152/files#diff-76da540a7b801708b1e0ebf6087f4100R12